js/global.js

```diff
@@ -823,7 +823,7 @@ jQuery(document).ready(function($) {
 
     gdn.stats = function() {
         // Call directly back to the deployment and invoke the stats handler
-        var StatsURL = gdn.url(gdn.getMeta('context')["dynamicPathFolder"] + '/settings/analyticstick.json');
+        var StatsURL = gdn.getMeta('context')["dynamicPathFolder"] + gdn.url('/settings/analyticstick.json');
```

Comment:
@egarciajulia  The code I see in this line breaks single responsibility principle.
Counter proposal here would be to either remove call to `gdn.url()`
```
var StatsURL = gdn.getMeta('context')["dynamicPathFolder"] + '/settings/analyticstick.json';
```
or pass this `dynamicPathFolder` to the function as second argument like:
```
var StatsURL =  gdn.url('/settings/analyticstick.json', gdn.getMeta('context')["dynamicPathFolder"]);
```

Reply:
Another thing to mention. I don't think that `gdn.getMeta('context')["dynamicPathFolder"] + '` is safe call to some predefined (we think) array key.

In case it is not defined, this code will produce string
```
StatsURL = 'undefined/settings/analyticstick.json'
```
with no error message or any exception thrown.